SUPPORT.md

```diff
@@ -1,13 +1,28 @@
 # Support
 
-## How to file issues and get help  
+## Microsoft Support Policy
 
-This project uses GitHub Issues to track bugs and feature requests. Please search the existing 
-issues before filing new issues to avoid duplicates.  For new issues, file your bug or 
-feature request as a new Issue.
+This repo is supported by Microsoft Support and users will be able to engage Microsoft support via their usual channels. Please provide corelation IDs where possible when contacting support to be able to investigate issue effectively and in timely fashion. For instruction on how to get deployments and correlation ID, please follow this link [here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deployment-history?tabs=azure-portal#get-deployments-and-correlation-id).
 
-For help and questions about using this project, please submit a Github issue with corresponding [Issue Labels found here](https://github.com/Azure/Enterprise-Scale/labels). Responses will be posted within 24 Hours. 
+Following list of issues are within the scope of Microsoft support:
 
-## Microsoft Support Policy  
+- Portal deployment of 4 Reference implementation - WingTip, Contoso and AdventureWorks, TreyResearch
```

Comment:
Let's remove Trey Research as it's not ready yet (readme is being worked on by FT)